code/modules/mob/living/silicon/robot/robot_damage.dm

```diff
@@ -107,9 +107,8 @@
 	if(status_flags & GODMODE)
 		return
 
-	if(damage_protection)
-		brute = clamp(brute - damage_protection, 0, brute)
-		burn = clamp(burn - damage_protection, 0, burn)
+	brute = clamp((brute - damage_protection)*brute_mod, 0, brute)
```

Comment:
This statement here makes it so that brute_mod can't be higher than 1 (if damage_protection is 0).
`X = clamp(X * 1.1, 0, X)` Min is 0, max is X so the result will be X.
A simple `max(X..., 0)` will do
Spacing as well here

Reply:
Done!